coalib/parsing/CliParsing.py

```diff
@@ -87,3 +88,27 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(sections):
```

Comment:
Are we not using the conflicting options thing ? https://docs.python.org/3/howto/argparse.html#conflicting-options